plugin/PDFDocumentPage.qml

```diff
@@ -40,19 +39,49 @@ DocumentPage {
         document: pdfDocument;
 
         onClicked: base.open = !base.open;
+
+        ViewPlaceholder {
+            enabled: pdfDocument.failure || pdfDocument.locked
+            y: (flickable ? flickable.originY : 0) + (base.height - height - (passwd.visible ? passwd.height : 0)) / 2
+            //% "Broken file" : "Locked file"
+            text: pdfDocument.failure ? qsTrId("sailfish-office-me-broken-pdf") : qsTrId("sailfish-office-me-locked-pdf")
+            //% "Cannot read the PDF document" : "Enter password to unlock"
+            hintText: pdfDocument.failure ? qsTrId("sailfish-office-me-broken-pdf-hint") : qsTrId("sailfish-office-me-locked-pdf-hint")
```

Comment:
Not sure if such syntax for translations works here normally it goes like:
     //% "Broken file"
    text: pdfDocument.failure ? qsTrId("sailfish-office-me-broken-pdf") : 
    //% "Locked file"
    qsTrId("sailfish-office-me-locked-pdf")